common/test/acceptance/pages/lms/account_settings.py

```diff
@@ -0,0 +1,61 @@
+"""
+Base class for account settings page.
+"""
+from . import BASE_URL
+
+from bok_choy.page_object import PageObject
+from bok_choy.promise import EmptyPromise
+
+from .fields import FieldsMixin
+
+
+class AccountSettingsPage(FieldsMixin, PageObject):
+    """
+    Tests for Account Settings Page.
+    """
+
+    url = "{base}/{settings}".format(base=BASE_URL, settings='account/settings')
+
+    def is_browser_on_page(self):
+        return 'Account Settings' in self.browser.title
+
+    def sections_structure(self):
+        """
+        Return list of section titles and field titles for each section.
+
+        Example: [
+            {
+                'title': 'Section Title'
+                'fields': ['Field 1 title', 'Field 2 title',...]
+            },
+            ...
+        ]
+        """
+        self.wait_for_ajax()
```

Comment:
self.wait_for_ajax command should never be required at the beginning of a method. 
Instead it should be put at the end of whatever action triggered an ajax call, like where you clicked on something.